### PR TITLE
Paystand Integration

### PR DESCRIPTION
**Pull request by elliotweaver**
Paystand Integration

### CODE REVIEW

**Comment by elliotweaver** [comment]
@andrewpaliga - Hi, can you code review what we have so far and offer some feedback? Thanks

File: lib/offsite_payments/integrations/paystand.rb [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+
+          request = Net::HTTP::Post.new(uri.path)
+
+          request['Content-Length'] = "#{payload.size}"
+          request['User-Agent'] = "Active Merchant -- http://activemerchant.org/"
+          request['Content-Type'] = "application/json"
+
+          http = Net::HTTP.new(uri.host, uri.port)
+          http.verify_mode    = OpenSSL::SSL::VERIFY_NONE unless @ssl_strict
+          http.use_ssl        = true
```
Comment (wvanbergen) [inline lib/offsite_payments/integrations/paystand.rb]: Please use `ActiveUtils::PostsData` to make HTTP(S) requests

File: test/unit/integrations/paystand/paystand_helper_test.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
+require 'test_helper'
+
+# sandbox api key
+PS_API_KEY = 'axr2RqZNFAJA9jelggHnmTXWzgPfBa6omk1OrsM0FIJfXRkvyS6oXDusTg9i70RRYJXxc+nmDNPttL2GfbAKV9A'
```
Comment (wvanbergen) [inline test/unit/integrations/paystand/paystand_helper_test.rb]: We should probably move this to `fixtures.yml`.

**Comment by cchen408** [comment]
@andrewpaliga - I've made the requested requested changes. Can you take a look and tell me what my next steps would be? Thank you!

**Comment by andrewpaliga** [comment]
@cchen408 on of the devs will review again once they have some time freed up. In terms of integration to shopify, please email payment-integrations@shopify.com with the info [here](https://github.com/Shopify/offsite-gateway-sim#getting-started) and why have you are not able to use our [Universal Gateway](https://github.com/Shopify/offsite_payments/blob/master/lib/offsite_payments/integrations/universal.rb) for integration (much faster route)

**Comment by pi3r** [comment]
I'll close this since it has been more than a year without activity

**Pull request closed without merging**